client/src/API/index.js

```diff
@@ -35,9 +35,12 @@ const updatePost = async (body, postId) => {
   }
 };
 
-const getCategories = async () => {
+const getCategories = async (token = undefined) => {
```

Comment:
if token is not provided then its by default undefined

Reply:
Whats wrong with this?

Reply:
nothing wrong, its just that it is obvious that token will be undefined if it is not provided